app/src/main/java/mobile/programming/musicApp/dto/Instrument.kt

```diff
@@ -1,11 +1,11 @@
-package mobile.programming.musicapp.dto
+package mobile.programming.musicApp.dto
 
 
 data class Instrument(var id: Int, var name: String, var type: String) {
 
     override fun toString(): String{
 
-        return (id.toString() + " " + name + " " + type)
+        return ("$id $name $type")
```

Comment:
Here, I replaced your string concatenation with a simple string template. It is apart of Kotlin coding convention is preferred throughout the program.